test_plans/queue-shortcut-functions.asciidoc

```diff
@@ -0,0 +1,100 @@
+:sectnums:
+:xrefstyle: short
+
+= Test plan for queue shortcut functions
+
+This is a test plan for the queue shortcut functions as described in SYCL 2020 Section 4.6.5.2.
+
+Estimated development time is four days.
+
+== Testing scope
+
+This plan does not include negative tests.
+
+=== Backend coverage
+
+All the tests described below are performed for any SYCL implementation.
+
+=== Device coverage
+
+All the tests described below are performed only on the default device that is selected on the CTS command line.
+
```

Comment:
What types `T` will be tested?  I think this should include at least one user-defined trivially copyable type (e.g. a struct with several fields).

Reply:
Good suggestion, I have added the user-defined struct and a list of other types to test for.